### PR TITLE
Dog Facts, Cat Facts, Birb Picture comands

### PR DESCRIPTION
**Pull request by ImCrispyy**
Hola, I am here with 3 commands, for the Fun module. Pretty please let me know if I made any mistakes, although I did follow the specified format completely.

Thanks Crispy outta here.

### CODE REVIEW

File: src/commands/Cfacts.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	constructor(...args) {
+		super(...args);
+
+		this.aliases        = ['catfact', 'catfacts']; // meow facts
+		this.module         = 'Fun';
+		this.description    = 'Get random Cat Facts with this commmand';
+		this.usage          = 'catfact';
+		this.example        = 'catfact';
+		this.cooldown       = 5000;
+		this.expectedArgs   = 1;
```
Comment (salesawy1) [inline src/commands/Cfacts.js]: `expectedArgs` should be 0 if you don't need args

File: src/commands/Dfatcs.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	constructor(...args) {
+		super(...args);
+
+		this.aliases        = ['dogfact', 'doggofact']; // woof facts
+		this.module         = 'Fun';
+		this.description    = 'Get random Dog Facts with this commmand';
+		this.usage          = 'dogfact';
+		this.example        = 'dogfact';
+		this.cooldown       = 5000;
+		this.expectedArgs   = 1;
```
Comment (salesawy1) [inline src/commands/Dfatcs.js]: same here

File: src/commands/Cfacts.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+	}	
+
+		async execute({ message }) {
+			try {
+				let res = await superagent.get('https://catfact.ninja/fact');
+				
+				return this.sendMessage(message.channel, res.body.value.joke);
```
Comment (salesawy1) [inline src/commands/Cfacts.js]: Also `res.body.value.joke` isn't valid. I just tested it, it'd be `res.body.fact`

File: src/commands/Dfatcs.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+	}	
+
+		async execute({ message }) {
+			try {
+				let res = await superagent.get('https://fact.birb.pw/api/v1/dog');
+				
+				return this.sendMessage(message.channel, res.body.value.joke);
```
Comment (Khaaz) [inline src/commands/Dfatcs.js]: Same here, just tested, the correct code would be:  `res.body.string`

File: src/commands/Bpics.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+			return msg.edit({
+				content: response.found,
+				embed: {
+					title: "https://random.birb.pw/tweet/random Tweet Tweet..",
+					color: 0x3498db,
+					image: {
+						url: res.redirects[0],
+					},
+					url: res.redirects[0],
```
Comment (Khaaz) [inline src/commands/Bpics.js]: the request here is incorrect. /tweet/random doesn't seems to work for me. Or at least no with resredirects.
It is enough to do =>
`let res = await request.get(`https://random.birb.pw/tweet`)`

But if you want to retrieve the image url you need to use res.text
Then to access the image the link would be:
`https://random.birb.pw/img/${res.text}`

res.redirects seems to just redirect to the input link.

Comment (Khaaz) [reply src/commands/Bpics.js]: Here a link.
https://cdn.discordapp.com/attachments/358674161566220288/415249368481005571/unknown.png

Ask me if you have questions :man_shrugging:

**Comment by briantanner** [comment]
On another note, file names should match class names, and ideally be readable names. i.e. Birb.js vs Bpics.js, and these commands need to be exported in `commands/index.js`

**Comment by briantanner** [comment]
Have to export the commands in `commands/index.js`

**Comment by ImCrispyy** [comment]
Done

**Pull request merged**